lib/httpPipeline.ts

```diff
@@ -124,6 +124,10 @@ export class HttpPipeline {
    * @return A Promise that resolves to the HttpResponse from the targeted server.
    */
   public send(request: HttpRequest): Promise<HttpResponse> {
+    if (!request || !(request instanceof HttpRequest)) {
```

Comment:
Just `!(request instanceof HttpRequest)` will do-- falsy values will return false for `instanceof HttpRequest`